enterprise_access/apps/content_assignments/tasks.py

```diff
@@ -131,3 +131,74 @@ def send_cancel_email_for_pending_assignment(cancelled_assignment_uuid):
         learner_content_assignment_action.traceback = exc
         learner_content_assignment_action.save()
         raise
+
+
+@shared_task(base=LoggedTaskWithRetry)
+def send_reminder_email_for_pending_assignment(assignment_uuid):
+    """
+    Send email via braze for reminding users of their pending assignment
+    Args:
+        assignment_uuid: (string) the subsidy request uuid
+    """
+    # importing this here to get around a cyclical import error
+    import enterprise_access.apps.content_assignments.api as content_api  # pylint: disable=import-outside-toplevel
+    learner_content_assignment_model = apps.get_model('content_assignments.LearnerContentAssignment')
+    subsidy_policy_model = apps.get_model('subsidy_access_policy.SubsidyAccessPolicy')
+    try:
+        assignment = learner_content_assignment_model.objects.get(uuid=assignment_uuid)
+    except learner_content_assignment_model.DoesNotExist:
+        logger.warning(f'request with uuid: {assignment_uuid} does not exist.')
+        return
+
+    try:
+        policy = subsidy_policy_model.objects.get(
+            assignment_configuration=assignment.assignment_configuration
+        )
+    except subsidy_policy_model.DoesNotExist:
+        logger.warning(f'policy with assignment config: {assignment.assignment_configuration} does not exist.')
+        return
+
+    learner_content_assignment_action = LearnerContentAssignmentAction(
+        assignment=assignment, action_type=AssignmentActions.REMINDED,
+    )
+    braze_trigger_properties = {}
+    braze_client_instance = BrazeApiClient()
+    lms_client = LmsApiClient()
+    enterprise_customer_uuid = assignment.assignment_configuration.enterprise_customer_uuid
+    enterprise_customer_data = lms_client.get_enterprise_customer_data(enterprise_customer_uuid)
+    admin_emails = [user['email'] for user in enterprise_customer_data['admin_users']]
+    course_metadata = content_api.get_content_metadata_for_assignments(
+        policy.catalog_uuid, assignment.assignment_configuration
+    )
+    lms_user_id = assignment.lms_user_id
+    braze_trigger_properties['contact_admin_link'] = braze_client_instance.generate_mailto_link(admin_emails)
+
+    try:
+        recipient = braze_client_instance.create_recipient(
+            user_email=assignment.learner_email,
+            lms_user_id=assignment.lms_user_id,
+        )
+        braze_trigger_properties["organization"] = enterprise_customer_data['name']
+        braze_trigger_properties["course_title"] = assignment.content_title
+        braze_trigger_properties["enrollment_deadline"] = course_metadata['normalized_metadata']['enroll_by_date']
+        braze_trigger_properties["start_date"] = course_metadata['normalized_metadata']['start_date']
+        braze_trigger_properties["course_partner"] = course_metadata['owners'][0]['name']
```

Comment:
This is only taking the first owner's name to send, is that an OK default to have? Or should I create a method that loops through if there's multiple and do "Harvard and Stanford" or something?

Reply:
I think that's fine for now.  We can circle back on this with relative ease in the future.

Reply:
Ah, that's a good point!